lua/lazyvim/config/init.lua

```diff
@@ -79,6 +79,8 @@ local defaults = {
 
 M.renames = {
   ["windwp/nvim-spectre"] = "nvim-pack/nvim-spectre",
+  ["jose-elias-alvarez/null-ls.nvim"] = "nvimtools/none-ls.nvim",
+  ["null-ls.nvim"] = "none-ls.nvim",
```

Comment:
Added these renames, so that users don't end up with both `null-ls` and `none-ls`, in case they reference `null-ls` in their own setup.

Reply:
Perfect